src/migrations/create.rs

```diff
@@ -72,6 +72,11 @@ pub struct Proposal {
     pub required_user_input: Vec<RequiredUserInput>,
 }
 
+// NOTE: clippy emits warning about needless question mark inside Queryable.String
+// This could be a bug in clippy.
+//
+// Adding #[allow(clippy::needless_question_mark)] does not help.
+#[allow(clippy::needless_question_mark)]
 #[derive(Deserialize, Queryable, Debug)]
```

Comment:
This is the only remaining clippy warning and it can't be silenced. Should be fixed in the original macros crate or I don't understand something.